cli/packages/util/init.go

```diff
@@ -0,0 +1,45 @@
+package util
+
+import (
+	"fmt"
+
+	"github.com/Infisical/infisical-merge/packages/api"
+	"github.com/Infisical/infisical-merge/packages/models"
+)
+
+func GetOrganizationsNameList(organizationResponse api.GetOrganizationsResponse) []string {
+	organizations := organizationResponse.Organizations
+
+	if len(organizations) == 0 {
+		message := fmt.Sprintf("You don't have any organization created in Infisical. You must first create a organization at %s", INFISICAL_TOKEN_NAME)
```

Comment:
Why is `INFISICAL_TOKEN_NAME` apart of the message?

Reply:
Should it be the `INFISICAL_DEFAULT_URL`

Reply:
<img width="2050" alt="Screenshot 2024-03-05 at 4 04 01 AM" src="https://github.com/Infisical/infisical/assets/47269183/7244f1e9-12f4-428b-9f95-116e43f9b64c">
I guess it was a mistake